src/OpenTelemetry.ResourceDetectors.AWS/AWSResourcesEventSource.cs

```diff
@@ -24,12 +24,12 @@ public void ResourceAttributesExtractException(string format, Exception ex)
     [Event(1, Message = "Failed to extract resource attributes in '{0}'.", Level = EventLevel.Warning)]
     public void FailedToExtractResourceAttributes(string format, string exception)
     {
-        this.WriteEvent(3, format, exception);
+        this.WriteEvent(1, format, exception);
```

Comment:
surprised that this was not caught in unit tests..!
@vishweshbankwar Can you check how? The main repo should be already running unit tests to cover these issues.. we should get it to run for contrib repo too. These issues makes it very tough to troubleshoot!